scss/partials/_cards.scss

```diff
@@ -1,12 +1,13 @@
 /** TABLE OF CONTENTS - Cards
  — — — — — — — — — — — — — — — — — — — — — — — — — — — — — — /
 1 — Content Block
+2 - Content Block - Clickable
 
 /// END TABLE OF CONTENTS **/
 
 /** 1 - Content Block
 ------------------------------------------------------------*/
-.content-block {
+.content-block, .content-block--clickable {
```

Comment:
Preferable selector here might be `[class^='content-block']`. This depends on what the nested selectors are, if they exist, but I'm assuming they don't if you can chain the initial selector like you did.